schs/schedules.bell

```diff
@@ -40,3 +40,26 @@
 14:08 Free
 
 * weekend # Weekend
+
+* holiday # Holiday
+
+* finals-wed # Wednesday Finals
+8:10 {Period 1}

```

Comment:
Need passing period for Period 1 and Period 7?

Reply:
Yes, I forgot to add them.